sdk/core/azure-core-amqp/src/impl/rust_amqp/amqp/session.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "azure/core/amqp/internal/session.hpp"
+
+#include "../../../models/private/performatives/detach_impl.hpp"
+#include "../../../models/private/value_impl.hpp"
+#include "azure/core/amqp/internal/connection.hpp"
```

Comment:
Broader question/curious about our guidelines: When do we typically have to use relative include paths like this, and when is it not okay to do?